layouts/_default/baseof.html

```diff
@@ -46,5 +46,8 @@ <h1>{{ .Site.Title }}</h1>
         {{ .Content }}
       {{ end }}
     </main>
+    <footer  class="app-footer">
+      {{ partial "footer.html" . }}
+    </footer>
```

Comment:
Not sure if it's better to make the footer integration optional. Would be happy about any guidance.

Reply:
My previous theme did this:

```
    <footer>
      {{ block "footer" . }}
      {{ end }}
    </footer>
```

Then you can define the footer in your other templates and pages.